package lab_9;

public class Main {
    public static void main(String[] args) {
        Exception e = new Exception();
        //e.exceptionDemo();
        //e.exceptionDemo1();
        //e.exceptionDemo2();
        //e.exceptionDemo3();
        ThrowsDemo throwsDemo = new ThrowsDemo();
        //throwsDemo.getDetails(null);
        //throwsDemo.getKey();
        //throwsDemo.getKey4();
    }
}




/*
e.exceptionDemo2();
Запустите эту программу со следующими выводами: Qwerty 0 1.2 1
Посмотрите на вывод. Какие исключения выбрасываются?
Измените код, добавив блоки try – catch, чтобы иметь дело с определяемыми исключениями.
*/

/*
e.exceptionDemo3();
И наконец добавьте блок finally к решению Задания №2. Повторно запустите программу, чтобы проверить ее поведение. Объясните новое поведение программы
*/
